client/src/components/profile/components/timeline-buttons.test.js

```diff
@@ -10,6 +10,18 @@ import Timeline from './time-line';
 Date.prototype.toLocaleString = jest.fn(() => 'Dec 29, 2022');
 Date.prototype.toISOString = jest.fn(() => '2016-09-28T20:31:56.730Z');
 
+const t = jest.fn(key => {
+  if (
+    Object.is(
+      key,
+      'intro:2022/responsive-web-design.blocks.learn-accessibility-by-building-a-quiz.title'
+    )
+  )
+    return 'Learn Accessibility by Building a Quiz';
+  else if (Object.is(key, 'aria.step')) return 'Step';
+  else return key;
+});
```

Comment:
I feel like this shouldn't need to be hardcoded. In fact, if I remove this locally (and the t prop down below), the tests pass for me (after updating the snapshot).

Reply:
I agree with you, because to tell the truth, after pushing this code, I noticed that it was not trying to solve the original issue but another one (the title disappears from the snapshot). I believe this pull request should only commit code focused on solving the original issue.